contracts/TimeManager.sol

```diff
@@ -0,0 +1,24 @@
+pragma solidity ^0.4.24;
+
+import "zeppelin-solidity/contracts/ownership/Ownable.sol";
+import "zeppelin-solidity/contracts/math/SafeMath.sol";
+
+contract TimeManager is Ownable {
+  using SafeMath for uint;
+
+  uint public lastRound;
+  uint public roundLength;
+  uint public rateLockDeadline;
```

Comment:
This is unused - we should either remove it or add a `TODO:` to add this functionality in later.

Reply:
You're right, I will remove it